tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -95,51 +95,51 @@ public static class ConfigurationKeys
         /// Overridden by <see cref="AgentUri"/> if present.
         /// Default value is "localhost".
         /// </summary>
-        /// <seealso cref="TracerSettings.AgentUri"/>
+        /// <seealso cref="TransportSettings.AgentUri"/>
```

Comment:
I feel like this could be a good opportunity to make `ConfigurationKeys` a partial class and move these keys out to a `ConfigurationKeys.Transport.cs` file